easybuild/easyblocks/p/python.py

```diff
@@ -143,7 +143,8 @@ def __init__(self, *args, **kwargs):
         self.pyshortver = '.'.join(self.version.split('.')[:2])
 
         # Used for EBPYTHONPREFIXES handler script
-        self.pythonpath = os.path.join(log_path(), 'python')
+        script_directory = 'etc'
+        self.pythonpath = os.path.join(script_directory, 'python')
```

Comment:
The variable is only used once and I don't understand the name: We put a script inside `pythonpath`, so why is there another `script_directory`?

```suggestion
        self.pythonpath = os.path.join('etc', 'python')
```

Reply:
I removed the extra variable as suggested.